src/biguint.rs

```diff
@@ -501,7 +501,7 @@ impl_shift! { i8, i16, i32, i64, i128, isize }
 impl Zero for BigUint {
     #[inline]
     fn zero() -> BigUint {
-        BigUint { data: Vec::new() }
+        BigUint::ZERO.clone()
```

Comment:
There's no need to `clone()` here. But since the `const` needs to be conditional anyway, let's just leave the original implementation as-is.  (Ditto on `BigInt`.)